src/discord/user.cpp

```diff
@@ -221,33 +221,33 @@ void UserData::update_from_json(const nlohmann::json &j) {
 
 const char *UserData::GetFlagName(uint64_t flag) {
     switch (flag) {
```

Comment:
can we just do this so we dont need a wall of static_casts ?
```cpp
    switch (static_cast<UserData::EFlags>(flag)) {
```

Reply:
For some reason I thought that this cast would be an undefined behavior. Yeah, that's actually much better

Reply:
Fixed.